app/views/layouts/_header.html.erb

```diff
@@ -46,7 +46,7 @@
         <li><a class="dropdown-button" href="#!" data-constrainwidth="false" data-beloworigin="true" data-activates="Donate">Donate<i class="material-icons right">arrow_drop_down</i></a></li>
 
         <li><%= link_to 'Success Stories', success_stories_path %></li>
-        <li><%= link_to 'Blog', 'https://medium.com/operation-code' %></li>
+        <li><%= link_to 'Blog', 'https://medium.com/operation-code', :target => "_blank" %></li>
```

Comment:
Can you please use the new style of hash, and single quotes around `_blank` since there's no interpolation occurring?